portfolio/src/main/webapp/index.html

```diff
@@ -4,13 +4,23 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="/script.js"></script>
   </head>
   <body>
     <div id="content">
       <h1>Jacinda's Portfolio</h1>
       <p>Welcome to my portfolio.</p>
       <img src = "/images/tenor.gif" />
+
+    <!--Button currently not connecting with url-->
+      <button onclick = "ShowSomething();">
```

Comment:
The capitalization here doesn't match - in script.js you don't capitalize the first s, but here you do. This might cause problems for you.

Reply:
Ah nvm, I see you fixed it in the next commit.